nms/app/packages/magmalte/app/views/equipment/EquipmentEnodeb.js

```diff
@@ -109,96 +116,140 @@ type EnodebRowType = {
 };
 
 function EnodebTableRaw(props: WithAlert) {
-  const {history, relativeUrl} = useRouter();
-  const ctx = useContext(EnodebContext);
-  const [currRow, setCurrRow] = useState<EnodebRowType>({});
-  const enbInfo = ctx.state.enbInfo;
+  const {history, relativeUrl, match} = useRouter();
   const enqueueSnackbar = useEnqueueSnackbar();
-  const enbRows: Array<EnodebRowType> = Object.keys(enbInfo).map(
-    (serialNum: string) => {
-      const enbInf = enbInfo[serialNum];
-      const isEnbManaged = enbInf.enb?.enodeb_config?.config_type === 'MANAGED';
-      return {
-        name: enbInf.enb.name,
-        id: serialNum,
-        numSubscribers: enbInf.enb_state?.ues_connected ?? 0,
-        sessionName: enbInf.enb_state?.fsm_state ?? '-',
-        ipAddress: enbInf.enb_state?.ip_address ?? '-',
-        mmeConnected: enbInf.enb_state?.mme_connected
-          ? 'Connected'
-          : 'Disconnected',
-        health: isEnbManaged ? (isEnodebHealthy(enbInf) ? 'Good' : 'Bad') : '-',
-        reportedTime: new Date(enbInf.enb_state.time_reported ?? 0),
-      };
-    },
+  const networkId: string = nullthrows(match.params.networkId);
+  const ctx = useContext(EnodebContext);
+  const [refresh, setRefresh] = useState(true);
+  const [lastRefreshTime, setLastRefreshTime] = useState(
+    new Date().toLocaleString(),
   );
 
+  // Auto refresh  every 30 seconds
+  const state = useRefreshingContext({
+    context: EnodebContext,
+    networkId: networkId,
+    type: 'enodeb',
+    interval: REFRESH_INTERVAL,
+    enqueueSnackbar: enqueueSnackbar,
```

Comment:
in case of an error, let's skip printing every 30 seconds. Retain the attribute in the prop, but you can remove it here.